omod/src/main/java/org/openmrs/web/controller/ConceptFormController.java

```diff
@@ -662,6 +668,11 @@ public Concept getConceptFromFormData() {
 				cn.setDisplayPrecision(displayPrecision);
 				cn.setUnits(units);
 				
+				for (ConceptReferenceRange referenceRange : this.referenceRanges) {
+					referenceRange.setConceptNumeric(cn);
```

Comment:
What leads to the state where a concept numeric is loaded without the reference ranges?

Reply:
An error will occur because of lack of foreign key.

Reply:
If you try run that code in debug mode, you will notice that the reference range already has the concept numeric property set.